ultra/ultra/evaluate.py

```diff
@@ -137,7 +142,7 @@ def evaluate(
 
         episode.reset(mode="Evaluation")
         while not dones["__all__"]:
-            action = agent.act(state, explore=False)
+            action = agent.act(state, explore=True)
```

Comment:
Oh interesting, are we allowing for exploration during evaluation?

Reply:
If generalization is the gap between training and testing performance, then we need to make sure we're comparing apples to apples. If we explore during training and not during testing, then exploration can conflate the difference in performance. The alternative is to run the agent on the train set without explorating during evaluation (which may be a better idea, but more time consuming)

Reply:
Oh okay yes that makes sense, thanks!